google-cloud-spanner/README.md

```diff
@@ -28,7 +28,7 @@ spanner = Google::Cloud::Spanner.new
 db = spanner.client "my-instance", "my-database"
 
 db.transaction do |tx|
-  results = tx.execute "SELECT * FROM users"
+  results = tx.execute_query "SELECT * FROM users"
```

Comment:
@blowmage - is there a possibility to leave this rename change out as per initial PR comment I had.
This is unrelated to DML / PDML and I rather not have a hiccup w/ the launch

Reply:
Yes. I can move it to a separate PR.